docs/getting-started/fundamentals/static-dynamic-rendering.md

```diff
@@ -0,0 +1,166 @@
+
+<details><summary><em>Example static rendering of <code>preformatted</code> core block</em></summary>
+<br/>
+For example, the following <a href="https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/preformatted/save.js"><code>save</code> function</a> of the <a hreh="https://github.com/WordPress/gutenberg/tree/trunk/packages/block-library/src/preformatted"><code>preformatted</code></a> core block...
+
+```js
+import { RichText, useBlockProps } from '@wordpress/block-editor';
+
+export default function save( { attributes } ) {
+	const { content } = attributes;
+
+	return (
+		<pre { ...useBlockProps.save() }>
+			<RichText.Content value={ content } />
+		</pre>
+	);
+}
+```
+
+...generates the following markup representation of the block when `attributes.content` has the value `"This is some preformatted text"`...
+
+```html
+<!-- wp:preformatted -->
+<pre class="wp-block-preformatted">This is some preformatted text</pre>
+<!-- /wp:preformatted -->
+```
+
+...and it will return the following markup for the block to the front end when there's a request.
+
+```html
+<pre class="wp-block-preformatted">This is some preformatted text</pre>
+```
+
+</details>
+
+<br/>
+
+Blocks with dynamic rendering can also define a markup representation of the block (via the `save` function) which can be processed in the server before returning the markup to the front end. If no dynamic rendering method is found, any markup representation of the block in the database will be returned to the front end.
+
+<div class="callout callout-info">
+The markup stored for a block can be modified before it gets rendered on the front end via hooks such as <a href="https://developer.wordpress.org/reference/functions/render_block/"><code>render_block</code></a> or via <code>$render_callback</code>.
+</div>
+
+Some examples of core blocks with static rendering (a.k.a. static blocks), this is blocks which output for the front end is statically generated when saved to the database (as returned by their `save` functions), are:
+- [`separator`](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/separator) (see its [`save`](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/separator/save.js) function) 
+- [`spacer`](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/spacer) (see its [`save`](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/spacer/save.js) function).
+- [`button`](https://github.com/WordPress/gutenberg/tree/trunk/packages/block-library/src/button) (see its [`save`](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/button/save.js) function).
+
+
+## Dynamic rendering
+
+Blocks with dynamic rendering are blocks that **build their structure and content on the fly when the block is requested from the client**. This type is block is often called a "dynamic block".
```

Comment:
"This type **of** block" instead of "This type **is** block."

Reply:
Fixed! ✅